cfgov/unprocessed/css/molecules/form-field-with-button.less

```diff
@@ -49,8 +49,8 @@
         margin-bottom: @margin__em / 2;
     }
 
-    .btn {
-        margin-bottom: @margin__em * 2;
+    div.short-desc {
```

Comment:
fixed in https://github.com/cfpb/cfgov-refresh/commit/127e98d41fd908bc46a3b3860431e3edc2edd179